service/collector/app_resource.go

```diff
@@ -134,7 +134,7 @@ func (c *AppResource) collectAppStatus(ctx context.Context, ch chan<- prometheus
 			app.Namespace,
 			app.Status.Release.Status,
 			team,
-			app.Status.Version,
+			app.Spec.Version,
```

Comment:
I can't think of any situations where we don't want to use this version.

What we really need is a list of status conditions like you specced in the observability changes.

But please add a comment so its clear why we get this from Spec not Status.

Reply:
Thanks, I will add a comment now.